srcpkgs/reproc/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'reproc'
+pkgname=reproc
+version=14.2.1
+revision=1
+build_style=cmake
+short_desc="Simplifies working with external CLI applications from C and C++"
+maintainer="teldra <teldra@rotce.de>"
+license="MIT"
+homepage="https://github.com/DaanDeMeyer/reproc"
+distfiles="https://github.com/DaanDeMeyer/reproc/archive/v${version}.tar.gz"
+checksum=f75f0524bdbf03813c126655f83d9c1b6a5b695c8e840bd62cd7aa61a96c66e3
+
+export CFLAGS="$CFLAGS -fPIC"
+export BUILD_CFLAGS="$BUILD_CFLAGS -fPIC"
```

Comment:
If you're adding to them, it should probably happen in `pre_configure`. And you can use `+=" -fPIC"`